packages/otelbin/src/components/monaco-editor/Editor.tsx

```diff
@@ -179,7 +179,7 @@ export default function Editor({ locked, setLocked }: { locked: boolean; setLock
 							)}
 						</div>
 						{viewMode !== "pipeline" && <ValidationErrorConsole errors={totalValidationErrors} font={firaCode} />}
-						{viewMode == "both" && <ResizeBar onWidthChange={onWidthChange} />}
```

Comment:
**Question:** Why was this condition removed? Now the resize bar is shown when the editor is hidden.